package/MDAnalysis/topology/TOPParser.py

```diff
@@ -620,3 +650,23 @@ def parse_dihedrals(self, diha, dihh):
         dihedrals = Dihedrals(dihed)
         impropers = Impropers(improp)
         return dihedrals, impropers
+
+    def parse_chainids(self, num_per_record, numlines):
+        """Extracts the chainID of each residue
+
+        Parameters
+        ----------
+        num_per_record : int
+            The number of entries for each record in section (unused input)
+        numlines : int
+            The number of lines to be parsed in current section
+
+        Returns
+        -------
+        attr : :class:`Segids`
+            A :class:`Segids` instance containing the chainID of each residue
+            as defined in the parm7 file
```

Comment:
Does this really return a segids instance?? It looks as if this just returns an array.

Reply:
Bad mistake. Fixed.